packages/oui-tile/_variables.less

```diff
@@ -0,0 +1,18 @@
+/* Tile */
+@oui-tile-background-color: #fff;
+@oui-tile-border-color: #dadce0;
+@oui-tile-border-radius: 3px;
```

Comment:
@neolitec : can you be more explicite ? :)

I think @antleblanc is right. Maybe you can define a @oui-tile-border using by default @oui-global-base-radius

```
@oui-tile-border-radius: @oui-global-base-radius;
```

It's true for other variables too :)